### PR TITLE
Update social meta tags for facebook and googleplus

### PR DESCRIPTION
**Pull request by hectoruch**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -22,6 +22,22 @@
...
+  <meta itemprop="name" content="Global Fishing Watch">
+  <meta itemprop="description" content="Global Fishing Watch enables anyone with an Internet connection to see fishing activity anywhere in the ocean in near real-time — for free">
+  <meta itemprop="image" content="https://raw.githubusercontent.com/Vizzuality/GlobalFishingWatch/master/public/apple-touch-icon.png">
+
+  <!-- Open Graph data -->
+  <meta property="og:url"                content="http://globalfishingwatch.org/" />
+  <meta property="og:type"               content="website" />
+  <meta property="og:title"              content="Global Fishing Watch" />
+  <meta property="og:description"        content="Global Fishing Watch enables anyone with an Internet connection to see fishing activity anywhere in the ocean in near real-time — for free" />
+  <meta property="og:image" content="https://raw.githubusercontent.com/Vizzuality/GlobalFishingWatch/master/public/apple-touch-icon.png" />
```
Comment (tiagojsag) [inline app/index.html]: change this to the actual prod url

File: app/index.html [hunk 2]
```diff
@@ -22,6 +22,22 @@
...
+  <meta property="og:image:secure_url" content="https://raw.githubusercontent.com/Vizzuality/GlobalFishingWatch/master/public/apple-touch-icon.png" />
```
Comment (tiagojsag) [inline app/index.html]: same here

File: app/index.html [hunk 3]
```diff
@@ -22,6 +22,22 @@
   <!-- http://stackoverflow.com/questions/4687698/mulitple-apple-touch-startup-image-resolutions-for-ios-web-app-esp-for-ipad -->
   <!-- <link rel="apple-touch-startup-image" href=""> -->
 
+  <!-- Schema.org markup for Google+ -->
+  <meta itemprop="name" content="Global Fishing Watch">
+  <meta itemprop="description" content="Global Fishing Watch enables anyone with an Internet connection to see fishing activity anywhere in the ocean in near real-time — for free">
+  <meta itemprop="image" content="https://raw.githubusercontent.com/Vizzuality/GlobalFishingWatch/master/public/apple-touch-icon.png">
```
Comment (tiagojsag) [inline app/index.html]: and here

**Comment by hectoruch** [comment]
Ready to review @tiagojsag

File: app/index.html [hunk 4]
```diff
@@ -18,10 +18,26 @@
   <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#2b4f93">
   <!-- this will run the website in "app" mode on iOs, adding a ~15s delay to open the website -->
   <!-- http://stackoverflow.com/questions/13413984/slow-initial-loadingscreen-ios-web-app -->
-  <!-- <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png"> -->
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```
Comment (tiagojsag) [inline app/index.html]: @nerik thoughts on uncommenting this line?

Comment (nerik) [inline app/index.html]: Meh. Yes.

**Pull request merged**